source/framework/core/src/TRestHits.cxx

```diff
@@ -852,7 +852,7 @@ Double_t TRestHits::GetDistanceToNode(Int_t n) const {
 }
 
 Int_t TRestHits::GetMostEnergeticHitInRange(Int_t n, Int_t m) const {
-    double maxEnergy = 0;
+    Int_t maxEnergy = 0;
```

Comment:
ups

Reply:
A commit like that should be in a stand-alone PR!

Reply:
Not in a "made some methods const"

Reply:
Well adding it back doesn't fix it. I think this bug or similar might be in more methods of the `TRestHits` class. Any ideas to fix the pipeline failure?

Reply:
The issue could also be at another library that was merged to master recently.

Reply:
so I think we can merge this into master right?

Reply:
Well, I think we should not merge unless we understand the issue. Or at least if the issue have been originated in this or related PR.

The pipeline for current master is passing?